plugins/container/src/caps/extract/extract.cpp

```diff
@@ -576,6 +577,7 @@ bool my_plugin::extract(const falcosecurity::extract_fields_input &in)
             thread_entry = m_threads_table.get_entry(tr, thread_id);
             // Retrieve container_id from the entry
             m_container_id_field.read_value(tr, thread_entry, container_id);
+            thread_entry_valid = true;
```

Comment:
I would push the `thread_entry` assignment out of this if/else to be able to cover more  cases

Reply:
well, probably it's worthless as the container events should not have a thread ID information associated.